app/src/components/AtomComponents/DateInputField.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+    <div class="input-field">
+      <label :for="id">{{ label }}:</label>
+      <input type="date" :id="id" :value="value" @input="updateValue" :required="isRequired" :max="maxDate" :min="minDate" />
+    </div>
+  </template>
+  
+  <script>
+  export default {
+    data(){
+        return{
+        minDate: new Date(new Date().getFullYear() - 100, 0, 1).toISOString().split("T")[0],
```

Comment:
Please avoid magic numbers in your code. I suggest assigning it to a special variable or adding a comment for this block of code.